extension/js/common/platform/xss.ts

```diff
@@ -64,8 +68,8 @@ export class Xss {
     return cleanHtml;
   }
 
-  public static htmlSanitizeAndStripAllTags = (dirtyHtml: string, outputNl: string): string => {
-    let html = Xss.htmlSanitizeKeepBasicTags(dirtyHtml);
+  public static htmlSanitizeAndStripAllTags = (dirtyHtml: string, outputNl: string, removeImgs: boolean = false): string => {
```

Comment:
Can you please share more info about how removeImgs is used, when and why? This would probably make for a good comment

Reply:
It's used because when the signature contains an image FlowCrypt won't render the image in the composer, it will instead render show image text, so I decided to remove Images from signature at all.
I used that here:
```TS
  private createFooterHTML = (footer: string) => {
    return `<br>-<br><br>${Xss.htmlSanitizeAndStripAllTags(footer, '<br>', true)}`;
  }
```

maybe when we have a WYSIWYG editor in FlowCrypt we could show images.

Reply:
Understood. I don't like footer images anyway, we don't have to support them.